solver.rb

```diff
@@ -0,0 +1,25 @@
+class Solver
+  def factorial(num)
+    if num.negative?
+      raise ArgumentError, 'Negative not allow'
+    elsif num.zero?
+      1
+    else
+      results = 1
+      (1..num).each { |a| results *= a }
+      results
+    end
+  end
+
+  def fizz_buzz(num)
+    return 'FizzBuzz' if (num % 3).zero? && (num % 5).zero?
+    return 'Fizz' if (num % 3).zero?
+    return 'Buzz' if (num % 5).zero?
+
+    num
```

Comment:
- [ ] @AbbasSarwar and @yegonkimutai great work adding fizz buzz method, however when a number is not divisible by 3 or 5 or both it should return a string of the number
![image](https://github.com/yegonkimutai/TDD/assets/61408860/2e9e5bb4-d907-48c3-a15e-4c04cca4ca56)